CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko
```

Comment:
`3.4` or `3.4.0`?

Reply:
we  have been going back and forth on this. 3.0, 3.1, 3.2.0, 3.30 happy to be consistent there.

Reply:
Let's try to follow the trend we established since `3.2.0`.
But I agree, for me `3.4.0` is the same as `3.4`